src/pipecat/services/cartesia.py

```diff
@@ -61,6 +62,14 @@ def language_to_cartesia_language(language: Language) -> str | None:
 
 
 class CartesiaTTSService(TTSService):
+    class InputParams(BaseModel):
+        model_id: Optional[str] = "sonic-english"
```

Comment:
I'd keep the `model_id` as a main argument as with all the other AI services. The rest looks great!

Reply:
There's also a conflict.